core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -271,6 +236,38 @@ public FolderId findFolder(FolderId baseFolderId, String folderName) throws File
 		}
 	}
 
+	public FolderId getBaseFolderId(String emailAddress, String baseFolderName) throws FileSystemException {
+		FolderId basefolderId;
```

Comment:
Zou je deze method onder configure() kunnen plaatsen, zodat in de GitHub-compare de code vergeleken wordt met open()?

Reply:
Done